sleet-api-service/src/main/java/com/sleet/api/service/QuoteService.java

```diff
@@ -2,16 +2,17 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.sleet.api.model.Equity;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import com.sleet.api.model.Equity;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.client.RestTemplate;
 
+import java.net.URL;
+import java.util.*;
```

Comment:
Might be pedantic, but would prefer to stick with explicit imports instead of wildcard

Reply:
No problem - IDE did this automatically.